packages/liveblocks-node/src/index.ts

```diff
@@ -33,6 +33,18 @@ export async function authorize(
   options: AuthorizeOptions
 ): Promise<AuthorizeResponse> {
   try {
+    const { room, userId, userInfo, secret } = options;
+
+    if (secret.startsWith("pk_")) {
+      throw new Error(
+        "Invalid key. You are using the public key which is not supported. Please use the secret key instead. For more information: https://liveblocks.io/docs/api-reference/liveblocks-node#authorize"
```

Comment:
I've been thinking about the language we use when we throw errors, and I think phrases like "Invalid key" or "You are […] which is not supported" may unnecessarily reinforce that the user made a mistake. To me, the golden standard of error messages is the Elm compiler, which is always very friendly and feels like a friend that helps you while setting things up or trying things out—and I think it's a great model to follow with all errors on Liveblocks.

In that style, we could consider erroring as follows:

_“We expect a secret key ("sk\_") here, but we found a public key ("pk\_") instead. Hint: You can find your secret key at https://liveblocks.io/dashboard/apikeys.”_

This doesn't put the blame on the user, but simply mentions what our expectations are, and hints at what the user could do next to fix this error.

I absolutely realize that our other existing error messages don't do this yet, but since we'd have to start somewhere, perhaps this is a good moment to start doing that here with these new error messages.